docs/ibc/light-clients/overview.md

```diff
@@ -1,3 +1,45 @@
 <!--
 order: 1
--->
\ No newline at end of file
+-->
+
+# Overview 
+
+Learn how to build IBC lightclient modules and fulfill the interfaces required to integrate with core IBC. {synopsis}
+
```

Comment:
Even though later on we assume that the reader has prior knowledge, I think it's best practice to concisely summarize what a light client does and what the context is it operates in (trust-minimized interoperability so we need light clients to track the consensus of a counterparty chain)